### PR TITLE
Update controller.rst

### PR DESCRIPTION
**Pull request by mervinmcdougall**
This update addresses some grammar issues and improves readability of the documentation.

### CODE REVIEW

File: controller.rst [hunk 1]
```diff
@@ -606,13 +606,16 @@ The ``file()`` helper provides some arguments to configure its behavior::
 Final Thoughts
 --------------
 
-Whenever you create a page, you'll ultimately need to write some code that
-contains the logic for that page. In Symfony, this is called a controller,
-and it's a PHP function where you can do anything in order to return the
-final ``Response`` object that will be returned to the user.
+In Symfony, a controller is usualll a class method which is used to accept requests,
```
Comment (javiereguiluz) [inline controller.rst]: Typo: `usualll` -> `usually`

File: controller.rst [hunk 2]
```diff
@@ -606,13 +606,16 @@ The ``file()`` helper provides some arguments to configure its behavior::
...
+and return a ``Response`` object. Symfony provides an  ``AbstractController`` which
+can be used to extend the controller class, and alllows access to some frequently used utilties
```
Comment (javiereguiluz) [inline controller.rst]: `alllows` -> `allows`

Comment (javiereguiluz) [inline controller.rst]: `utilties` -> `utilities`

File: controller.rst [hunk 3]
```diff
@@ -606,13 +606,16 @@ The ``file()`` helper provides some arguments to configure its behavior::
...
+In Symfony, a controller is usualll a class method which is used to accept requests,
+and return a ``Response`` object. Symfony provides an  ``AbstractController`` which
+can be used to extend the controller class, and alllows access to some frequently used utilties 
+such as ``render`` and ``redirectToRoute``. The ``AbstractController`` also 
+provides the ``createNotFoundException`` utility which is used to return a page
+not found response.
 
-To make life easier, you'll probably extend the base ``AbstractController`` class because
-this gives access to shortcut methods (like ``render()`` and ``redirectToRoute()``).
+Services are accesible in controllers by typehinting them in the arguments of the controller methods.
```
Comment (javiereguiluz) [inline controller.rst]: I don't like this change ... I think it's too early to talk about services. The previous text, which mentioned but didn't explain services looked better to me.

**Review by javiereguiluz — COMMENTED** [review]
@mervinmcdougall thanks for this contribution. I like most of your changes a lot ... but I have some concerns about the changes made at the bottom of the article.

File: controller.rst (hunk 1 above)
Comment (mervinmcdougall) [reply controller.rst]: The typo identified below has been addressed.
```suggestion
In Symfony, a controller is usualll a class method which is used to accept requests,
```

File: controller.rst (hunk 2 above)
Comment (mervinmcdougall) [reply controller.rst]: This has been reworded in the most recent commit.

Comment (mervinmcdougall) [reply controller.rst]: This typo identified below has been addressed.
```suggestion
can be used to extend the controller class, and alllows access to some frequently used utilties 
```

File: controller.rst (hunk 3 above)
Comment (mervinmcdougall) [reply controller.rst]: I  have removed the paragraph which describes services.

**Comment by javiereguiluz** [comment]
Thank you @mervinmcdougall. This has been finally merged in 4.2 branch (the oldest maintained branch where it applies). We also merged in 4.3, 4.4 and master. Cheers!

**Pull request merged**